lib/elastomer/client/docs.rb

```diff
@@ -597,6 +597,7 @@ def update_params(params, overrides = nil)
         h = defaults.update params
         h.update overrides unless overrides.nil?
         h[:routing] = h[:routing].join(",") if h[:routing].is_a?(Array)
+        if client.version_support.es_version_7_plus? then h[:type] = "_doc" end
```

Comment:
Minor, but I typically see similar Ruby one-liners written this way:
```
h[:type] = "_doc" if client.version_support.es_version_7_plus?
```

Reply:
Done, thanks!